tools/privacyidea-cron

```diff
@@ -95,6 +95,9 @@ def run_task_on_node(ptask, node):
         print_stderr(u'Task {!r} on node {!r} did not run '
                      u'successfully.'.format(ptask["name"], node))
         print_stderr(u'This unsuccessful run is not recorded in the database.')
+        if not ptask["retry_if_failed"]:
```

Comment:
I would write this as

~~~~python
if not ptask.get("retry_if_failed"):
~~~~

If there is any corner case where `ptask` is missing the element "retry_if_failed", this would not result in an exception.

Reply:
Done in 969450a5